app/javascript/styles/_keywordSearch.scss

```diff
@@ -64,3 +64,10 @@
 .gene-keyword-search-input {
   flex: 1;
 }
+
+.study-explore .gene-keyword-search {
+	width: 100%;
+	.input-group {
+		width: 100%;
+	}
+}
```

Comment:
Spacing is off.

Reply:
yeah, something got wonky with my sublime, will attempt to fix...